sample_app/main_rtk.c

```diff
@@ -121,28 +121,40 @@ static int _cmd_pnio_alarm_ack(
 
 const shell_cmd_t cmd_pnio_alarm_ack =
 {
-      .cmd = _cmd_pnio_alarm_ack,
-      .name = "pnio_alarm_ack",
-      .help_short = "Send AlarmAck",
-      .help_long ="Send AlarmAck"
+   .cmd = _cmd_pnio_alarm_ack,
+   .name = "pnio_alarm_ack",
+   .help_short = "Send AlarmAck",
+   .help_long ="Send AlarmAck"
 };
 SHELL_CMD (cmd_pnio_alarm_ack);
 
 
 static int _cmd_pnio_show(
-      int                  argc,
-      char                 *argv[])
+   int                     argc,
+   char                    *argv[])
 {
-   unsigned long level = 0;
-   char *end = NULL;
+   unsigned long           level = 0x2010;  /*  See documentation for pnet_show()  */
+   char                    *end = NULL;
 
-   if (argc == 2)
+   if (argc > 2)
+   {
+      printf("Usage: pnio_show level\n");
+      return -1;
+   }
+   else if (argc == 2)
    {
       level = strtoul(argv[1], &end, 0);
    }
+   if (level < 1)
```

Comment:
```suggestion
   if (end == argv[1] || level == 0)
```

Reply:
Changed to 
`if (level == 0)`
as we should allow the use of a default value.